apps/dsp-app-e2e/cypress/support/commands.ts

```diff
@@ -0,0 +1,62 @@
+/// <reference types="cypress" />
+
+import { User } from "../models/user-profiles";
+
+// ***********************************************
+// This example commands.ts shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
```

Comment:
You could speed up tests by here just calling the api like /login, and get the auth token instead of going through with the UI.